webview/src/experiments/components/table/body/ExperimentStatusIndicator.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { VSCodeProgressRing } from '@vscode/webview-ui-toolkit/react'
+import cx from 'classnames'
+import {
+  ExperimentStatus,
+  GitRemoteStatus,
+  isRunning
+} from 'dvc/src/experiments/webview/contract'
+import { CellHintTooltip } from './CellHintTooltip'
+import styles from '../styles.module.scss'
+import { clickAndEnterProps } from '../../../../util/props'
+import { pushExperiment } from '../../../util/messages'
+import { Cloud, CloudUpload } from '../../../../shared/components/icons'
+import { Icon } from '../../../../shared/components/Icon'
+
+type ExperimentStatusIndicatorProps = {
+  status: ExperimentStatus | undefined
+  gitRemoteStatus: GitRemoteStatus | undefined
+  id: string
+}
+
+export const ExperimentStatusIndicator: React.FC<
+  ExperimentStatusIndicatorProps
+> = ({ id, status, gitRemoteStatus }) => {
+  if (isRunning(status) || gitRemoteStatus === GitRemoteStatus.PUSHING) {
```

Comment:
[I] This `status` field should be renamed to `executorStatus` or similar in a follow up